spec/syntax.md

```diff
@@ -526,18 +528,73 @@ unrecognized reserved sequences have no meaning and MAY result in errors during
 
 ```abnf
 reserved       = reserved-start reserved-body
-reserved-start = "!" / "@" / "#" / "%" / "*" / "<" / ">" / "/" / "?" / "~"
+reserved-start = "!" / "#" / "%" / "*" / "<" / ">" / "/" / "?" / "~"
 
 reserved-body  = *( [s] 1*(reserved-char / reserved-escape / quoted))
 reserved-char  = %x00-08        ; omit HTAB and LF
                / %x0B-0C        ; omit CR
                / %x0E-19        ; omit SP
-               / %x21-5B        ; omit \
+               / %x21-3F        ; omit @
+               / %x41-5B        ; omit \
                / %x5D-7A        ; omit { | }
                / %x7E-D7FF      ; omit surrogates
                / %xE000-10FFFF
 ```
 
+### Expression Attributes
+
+An **_<dfn>expression attribute</dfn>_** is a key-value pair
+containing a named argument that is attached to the _expression_ as a whole.
+They are intended to convey information about the _expression_
+that is not necessarily applicable as a _function_ _option_.
+For example, the `@locale` attribute overrides the locale of the _message_ for its _expression_.
+Attributes like `@can-delete` and `@translate` might not have any impact on formatting,
+acting instead as instructions to translators.
```

Comment:
The syntax specification should only include attributes that we formally define all the way through. If these are examples, they should be treated as such, e.g. "If an implementation supported attributes such as..."